Hometask8/src/main/java/homework21/LoginPage.java

```diff
@@ -3,8 +3,11 @@
 import static com.codeborne.selenide.Selenide.$;
 
 public class LoginPage {
-    public static void login(String username, String password) {
-        $("#user-name").setValue(username);
+    static final String userName = "standard_user";
+    static final String password = "secret_sauce";
```

Comment:
test data should not be stored in Page object, pass it from test methods to page object methods